plugins/CoreHome/vue/dist/CoreHome.umd.js

```diff
@@ -6174,7 +6162,8 @@ function debounce(fn) {
       type: String,
       default: 'bottom'
     },
-    placeholder: String
+    placeholder: String,
+    defaultToFirstSite: Boolean
```

Comment:
doesn't that one need a default value? Or is that automatically `false` for booleans?

Reply:
If a default value isn't supplied, or `required: true` isn't set, the default value will be `undefined`. Since we just check if its truthy, its fine for booleans.